documentation/Versioning and changelog.md

```diff
@@ -97,3 +97,35 @@ Because we expose both React components (for which the markup, including class n
 - Changes that do not impact public APIs
 - Non-breaking changes to minimum version of dependencies
 - Breaking changes to private Sass variables, functions, and mixins
+
+## Creating a new release
+
+Follow the following steps to deploy a new version of Polaris:
```

Comment:
This has to be done by a @shopify employee. We should probably add that as a disclaimer.